packages/app/components/map/WebMap.web.tsx

```diff
@@ -1,15 +1,12 @@
 import React, { useState } from 'react';
 import mapboxgl from 'mapbox-gl';
-import { MAPBOX_ACCESS_TOKEN } from '@env';
```

Comment:
we should use this instead of hardcoded key below, did not realize that was in there lol